cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java

```diff
@@ -250,6 +253,21 @@ private String guessOrgName() {
         return guessOrgName;
     }
 
+    /**
+     * Validates the org-name.
+     *
+     * @param orgName The org-name.
+     * @return True if valid org-name, else false.
+     */
+    private boolean validateOrgName(PrintStream out, String orgName) {
+        boolean matches = RepoUtils.validateOrg(orgName);
+        if (!matches) {
+            out.println("--Invalid organization name: \'" + orgName + "\'. Organization name can only contain " +
```

Comment:
Why does this message start with two hyphens? Also, is this writing to stdout or stderr?

Reply:
Warnings in the init command are started with two hyphens. This is writing to stdout and not to stderr since this is not an error but actually a warning to the user.